kubecost/values.yaml

```diff
@@ -54,58 +54,24 @@ global:
 
   ## Kubecost Alerting
   ## Ref: https://www.ibm.com/docs/en/kubecost/self-hosted/2.x?topic=ui-alerts
-  ##
   notifications:
     # alertConfigs:
-    #   frontendUrl: http://localhost:9090  # Optional
+    #   frontendUrl: http://localhost:9090  # Optional. When configured, the alert message includes a link to the page you want to visit for analyzing the alert.
```

Comment:
"the page" or just the URL of the Kubecost instance's overview? Why/when would this be configured by a user?